server/lib/sightingsController.js

```diff
@@ -11,8 +11,16 @@ const logger = require('./logger.js');
 // Get all sightings in the database
 exports.getSightings = (searchOpts = {}) => {
 	logger.info('Getting Sightings...');
-	logger.info(`Using some variables so lint doesn't scream: ${searchOpts}`);
-	return Sighting.find({});
+	const showLimit = 5;
+	let skips = 0;
+
+	if (searchOpts.page) {
```

Comment:
This will need to get moved into query builder & tested ofc

Reply:
Would you normally merge as is, and then in the profile branch update it? Or update it now before merge